sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -783,6 +799,8 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "currentAssessment.assessmentWorkflowId",
       ])
       .innerJoin("application.programYear", "programYear")
+      .innerJoin("application.parentApplication", "parentApplication")
+      .innerJoin("application.precedingApplication", "precedingApplication")
```

Comment:
Minor. From the perspective of the method, I would suggest to use a left join rather.